code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -73,4 +73,16 @@
 
 /turf/simulated/floor/carpet/burn_tile()
 	burnt = 1
-	update_icon()
\ No newline at end of file
+	update_icon()
+
+
+
+/turf/simulated/floor/fakespace
+	icon = 'icons/turf/space.dmi'
+	icon_state = "0"
+	floor_tile = /obj/item/stack/tile/fakespace
+	broken_states = list("damaged")
+
+/turf/simulated/floor/fakespace/New()
+	..()
+	icon_state = "[rand(0,25)]"
```

Comment:
I thought about doing it that way but it doesn't really make sense for carpeting to exactly mirror the underlying space. Maybe if they were were glass-bottom stations or something that would be good.